src/Features/OnboardingAutomateTaxes.php

```diff
@@ -58,13 +59,7 @@ public static function on_onboarding_profile_completed() {
 		if ( $jetpack_connected && $wcs_version && $wcs_tos_accepted ) {
 			update_option( 'wc_connect_taxes_enabled', 'yes' );
 			update_option( 'woocommerce_calc_taxes', 'yes' );
+			Confirm_Tax_Settings::possibly_add_note();
 		}
 	}
-
-	/**
-	 * Check if automated taxes are supported.
-	 */
-	private static function automated_tax_is_supported() {
```

Comment:
It was a bug that I didn't call this method in my original implementation. I've added it in #5276

Reply:
ok I'll add this back